doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -658,7 +665,11 @@ Matter samples
 
   * :ref:`ug_matter_configuring_read_client` in most Matter samples using the new :kconfig:option:`CONFIG_CHIP_ENABLE_READ_CLIENT` Kconfig option.
   * WPA Supplicant advanced features in all Matter samples using the :kconfig:option:`WPA_SUPP_ADVANCED_FEATURES` Kconfig option.
-    This saves roughly 25 KB of FLASH memory for firmware images with Wi-Fi support.
+    This saves roughly 25 kB of FLASH memory for firmware images with Wi-Fi support.
+
+* :ref:`matter_lock_sample` sample:
+
+  * Added support for Wi-Fi firmware upgrade on external memory.
```

Comment:
This is valid only for `nrf5340dk_nrf5340 + nrf7002EK`, so could you mention it somewhere?

The second question, could you please add a similar note to `Matter Bridge` Application? Regarding that application, I've added support for `nrf5340dk_nrf5340 + nrf7002EK` (Previously we had support for `nrf7002dk_nrf5340` only), and now we have the same support for Wi-Fi firmware upgrade on external memory there (also valid for  `nrf5340dk_nrf5340 + nrf7002EK` only )

Reply:
Added, please confirm if it is correctly described.